### PR TITLE
[Deployment] Node upload yaml fix

### PR DESCRIPTION
**Pull request by pkommoju**
This PR has three changes:

1) Corrections to NCM yaml file (k8s)
2) Delete command in Services (k8s)
3) A fix which disappeared (either as a mixup or somehow missed merging)

### CODE REVIEW

**Review by cj-chung — APPROVED** [review]
LGTM

File: docs/modules/ROOT/pages/db_services/alcor-transactions.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
+== Transactions in Alcor
+Prasad Kommoju <pkommoju@futurewei.com>
+
+v.01, 2021-05-05
+
+:toc: right
+
+== Overview
```
Comment (xieus) [inline docs/modules/ROOT/pages/db_services/alcor-transactions.adoc:8]: @pkommoju Is this file included in other PR or merged to master already?

Comment (pkommoju) [reply docs/modules/ROOT/pages/db_services/alcor-transactions.adoc:8]: It has been merged but I think I have made these changes after the merge.

Comment (xieus) [reply docs/modules/ROOT/pages/db_services/alcor-transactions.adoc:8]: Actually the file alcor-transactions.adoc is under review in PR #611. Let us remove it from this PR and get it reviewed in PR #611

Comment (pkommoju) [reply docs/modules/ROOT/pages/db_services/alcor-transactions.adoc:8]: I am having git push trouble. I had sent a query to github, got a reply. I am try it now.

**Comment by pkommoju** [comment]
Abandon PR to remove spurious update to alcor-transactions.adoc which has its own PR ( #611)

**Pull request closed without merging**